interface-definitions/nat_cgnat.xml.in

```diff
@@ -8,6 +8,12 @@
           <priority>221</priority>
         </properties>
         <children>
+          <leafNode name="log-allocation">
```

Comment:
Why not simply use `log` like we have in firewall, nat44, nat66?

Reply:
`log-allocation` vs `log` is different
The log in CGN means logging the sessions that are not implemented
i.e. session internal IP <x.x.x.x> sport <xx> destination address <y.y.y.y> port <yy> protocol <proto>

Allocation just says which internal IP is allocated to which external IP and which external port per commit